test/plugins/features/movies/controller.test.js

```diff
@@ -23,4 +24,52 @@ describe('movie controller', () => {
 
   });
 
+  describe('list', () => {
+
+    beforeEach(() => {
+      const movieOne = { title: 'Batman Begins', release_year: 2005 };
+      const movieTwo = { title: 'The Dark Knight', release_year: 2008 };
+      const movieThree = { title: 'The Dark Knight Rises', release_year: 2012 };
+
+      return Knex('movies').truncate()
+      .then(() => {
+        return Knex('movies').insert([movieOne, movieTwo, movieThree]);
+      });
+    });
+
+    it('lists all movies', () => {
+      const query = {};
+
+      return Controller.list(query)
+      .then((movies) => {
+        expect(movies).to.have.lengthOf(3);
+      });
+    });
+
+    it('filters by title', () => {
+      const query = { title: 'the dark knight' };
+
+      return Controller.list(query)
+      .then((movies) => {
+        expect(movies).to.have.lengthOf(1);
+
+        movies.each((movie) => {
```

Comment:
Nit: we typically use `forEach`. It does the same exact thing, but consistency is usually a good thing in team settings.